packages/core/contracts-ovm/insured-bridge/implementation/BridgeDepositBox.sol

```diff
@@ -182,22 +188,31 @@ abstract contract BridgeDepositBox is Legacy_Testable, Legacy_Lockable {
             getCurrentTime() >= quoteTimestamp - 10 minutes && getCurrentTime() <= quoteTimestamp + 10 minutes,
             "deposit mined after deadline"
         );
+        // If the address of the L1 token is the l1Weth and there is a msg.value with the transaction then the user
```

Comment:
Just so I understand, does this mean that you set all the params like you're sending WETH, you just provide a msg.value and the contract will just not pull WETH from you?

Reply:
correct. you do everything the same as weth and include a `value` and it will use eth.